### PR TITLE
DB-370: Fix branding system work on Windows

### PR DESCRIPTION
**Pull request by alver-cliqz**
Command line was not correctly processing in case parent process use switches.

### CODE REVIEW

File: mozilla-release/other-licenses/nsis/Contrib/CliqzHelper/helper.cpp [hunk 1]
```diff
@@ -130,3 +118,20 @@ std::wstring GetParentCommandString() {
...
+  if (pid != 0) {
+    TCHAR filename[MAX_PATH];
+
+    HANDLE processHandle = OpenProcess(
+        PROCESS_QUERY_INFORMATION | PROCESS_VM_READ, FALSE, pid);
+    if (processHandle != NULL) {
+      if (GetModuleFileNameEx(processHandle, NULL, filename, MAX_PATH)) {
+        cmd = filename;
+      }
+      CloseHandle(processHandle);
```
Comment (maxim-cliqz) [inline mozilla-release/other-licenses/nsis/Contrib/CliqzHelper/helper.cpp]: See `mozilla-release/security/sandbox/chromium/base/win/scoped_handle.h`

File: mozilla-release/other-licenses/nsis/Contrib/CliqzHelper/installer_tagdata.cpp [hunk 2]
```diff
@@ -53,8 +53,10 @@ bool InstallerTagData::Init(const std::wstring& cmd) {
 
 // static
 void InstallerTagData::Reset() {
-  delete current_installer_tagdata_;
-  current_installer_tagdata_ = NULL;
+  if (current_installer_tagdata_ != NULL) {
+    delete current_installer_tagdata_;
+    current_installer_tagdata_ = NULL;
+  }
```
Comment (maxim-cliqz) [inline mozilla-release/other-licenses/nsis/Contrib/CliqzHelper/installer_tagdata.cpp]: Why not use `std::unique_ptr`?

**Comment by maxim-cliqz** [comment]
Except where noted, looks OK.

File: mozilla-release/other-licenses/nsis/Contrib/CliqzHelper/helper.cpp (hunk 1 above)
Comment (alver-cliqz) [inline mozilla-release/other-licenses/nsis/Contrib/CliqzHelper/helper.cpp]: This is not "mozilla based" project, it don't have any dependencies with mozilla source code.

File: mozilla-release/other-licenses/nsis/Contrib/CliqzHelper/installer_tagdata.cpp (hunk 2 above)
Comment (alver-cliqz) [inline mozilla-release/other-licenses/nsis/Contrib/CliqzHelper/installer_tagdata.cpp]: Too much for rewrite here. Also will add needs in C++11 standard, while we have nothing from it in this project. I think we can process without it for now, but I will count this moment on next refactoring or bugfixing.

Comment (xzxzzxzxzca) [inline mozilla-release/other-licenses/nsis/Contrib/CliqzHelper/installer_tagdata.cpp]: `std::auto_ptr` is available :)
just saying

**Pull request merged**